app/forms/concerns/work_form_behavior.rb

```diff
@@ -68,6 +68,16 @@ def model_attributes(params)
       #  -- they don't have these fields in the form at all so they don't get repopulated.
       clean_params = encode_physical_container(params, clean_params)
       clean_params = encode_external_id(params, clean_params)
+
+      clean_params.keys.each do |key|
```

Comment:
The example from awead did it after super, and your existing transformations were after super too! So after super seemed the way to go. 

I have no real opinion on whether extracting to a method in this case will make it more or less readable/maintainable. But I plan to add more stuff to the `clean_params.keys.each` loop (for description formatting), so I want to leave that loop in the main method. 

What do you think about waiting until we add a bit more stuff, and then refactoring? I also wasn't sure what the existing stuff did exactly, so wasn't sure how to refactor it.